lib/package-initialization-error-component.js

```diff
@@ -16,12 +16,27 @@ class PackageInitializationErrorComponent {
   render () {
     return $.div({className: 'PackageInitializationErrorComponent'},
       $.h3(null, 'Teletype initialization failed'),
-      $.p(null, 'Make sure your internet connection is working and restart the package.'),
+      $.p(null, 'Make sure your internet connection is working and reload Atom.'),
+      $.div(null,
+        $.button(
+          {
+            ref: 'reloadButton',
+            type: 'button',
+            className: 'btn btn-primary inline-block-tight',
+            onClick: this.reloadAtom
+          },
+          'Reload Atom'
+        )
+      ),
       $.p(null,
         'If the problem persists, visit ',
         $.a({href: 'https://github.com/atom/teletype/issues/new', className: 'text-info'}, 'atom/teletype'),
         ' and open an issue.'
       )
     )
   }
+
+  reloadAtom () {
+    location.reload() // eslint-disable-line no-undef
```

Comment:
@as-cii: Is this a reasonable approach? Or is there another implementation that would be more preferable here?

Reply:
You can also call `atom.reload()`, which RPCs back to the main process. I'm not actually clear on whether there ends up being a material difference. The same exact code may end up running on reload either way. We'd have to investigate to be totally sure.